projects/estonia/docker-compose.yml

```diff
@@ -116,7 +116,7 @@ services:
     security_opt:
       - seccomp=unconfined
   pelias-gtfs:
-    image: dolmit/pelias-gtfs:latest
+    image: peatusee/pelias-gtfs:latest
```

Comment:
FYI we have the [pelias/transit](https://github.com/pelias/transit) importer which can also import GTFS stops.

Does this package offer something that's missing from `pelias/transit`?

Reply:
Ping @iSudo

Reply:
hi, yes I do remember trying out the pelias/transit for gtfs, but it did not work for us that time, may be it was because we had differences in dataset, i can't remember, but I can assure I tryed to make it work with pelias/transit, and failed